### ISSUE TITLE
RECOMMENDATION: Use the Project Page to list out the To-Dos

### ISSUE THREAD

**Issue by BrandonKMLee** [alberanid/imdbpy#335]
These are some tools that are useful to deal with the current lack of features in IMDBPy
- https://docs.github.com/en/discussions
- https://github.com/features/issues

**Comment by alberanid** [comment]
Makes sense; apparently they are beta features, but we can try to use them (not really familiar with the Github implementation of these features)

**Comment by BrandonKMLee** [comment]
Hope that this can be done in  https://github.com/alberanid/imdbpy/projects
